aws/sdk/sync-models.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import os
+from pathlib import Path
+import shutil
+
+if len(sys.argv) != 2:
+    print("Please provide the location of the aws-models repository as the first argument")
+    sys.exit(1)
+
+aws_models = sys.argv[1]
+
+for model in os.listdir("aws-models"):
```

Comment:
Generally a good idea to change directory into the script's directory just to be sure this will actually exist in case someone runs the script from the repository root.